src/org/yccheok/jstock/engine/Index.java

```diff
@@ -56,7 +56,8 @@
     TWII(Country.Taiwan, "TSEC", "Taiwan Stock Exchange Weighted Index", Code.newInstance("^TWII")),
     FTSE(Country.UnitedKingdom, "FTSE 100", "FTSE 100 Index", Code.newInstance("^FTSE")),
     DJI(Country.UnitedState, "DOW JONES", "Dow Jones Industrial Average", Code.newInstance("^DJI")),
-    IXIC(Country.UnitedState, "NASDAQ", "NASDAQ Composite Index", Code.newInstance("^IXIC"));
+    IXIC(Country.UnitedState, "NASDAQ", "NASDAQ Composite Index", Code.newInstance("^IXIC")),
+    SP500(Country.UnitedState, "S&P500", "Standard and Poors 500 Index", Code.newInstance("^GSPC"));
```

Comment:
Well, any finance report in the US will show all the 3 indexes - S&P, Nasdaq and DOW. If you look at how each is compiled (its origins) I would argue that the S&P 500 is the best "general consensus" of the US market. The DOW is basically mega-cap and too small a sample (40 companies?), the Nasdaq is overly tech-focused. Leaving the S&P as the best general indicator. But I agree that typically "less is more" in good software development.